exercises/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,21 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+function spellNameUpperCase(firstName) {
+    return firstName.toUpperCase();
+}
+
+function createShoutyGreeting(firstName) {
+    var message = "HELLO " + spellNameUpperCase(firstName);
+    return message;
+}
+
+console.log (createShoutyGreeting("Daniel"));
```

Comment:
Hi Yasemin, It would be better to use variable names as the function parameter in console.log
your code is nicely indented. well done

Reply:
Thank you very much. I have changed this.